### PR TITLE
Trigger events once

### PR DESCRIPTION
**Pull request by jvillafanez**
## Description
Prevent the registration of the same event listener several times, which causes the same action (from the duplicated listeners) to be performed multiple times.

Primary problem is with the "urlChanged" event, which causes an ajax request to be thrown. This means that, without this fix, the ajax requests performed were increasing along with the increasing event listeners being registered for that event.
The other events don't seem to be too critical since the attached action is pretty light.

## Related Issue
Helps with https://github.com/owncloud/enterprise/issues/4374

## Motivation and Context
No need to register the same event listener twice

## How Has This Been Tested?
Mostly tested going back and forth between the "all files" and "shared with you" sections in the files view, as well as checking there is no apparent problem with the scrolling in those sections.
Event registration has been checked with the firefox dev tools, ensuring there is no increasing event associated with the target web element. There could be multiple listeners for the same event, but it's expected to be for different purposes (not duplicated)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [x] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Review by JammingBen — APPROVED** [review]
Didn't know that namespacing jQuery-events is a thing, cool stuff :+1:

**Comment by karakayasemi** [comment]
Looks good. Fixes https://github.com/owncloud/core/issues/37085

File: changelog/unreleased/38385 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Bugfix: Prevent duplicated calls by not registering the same listener twice
```
Comment (JanAckermann) [inline changelog/unreleased/38385:1]: duplicated -> multiple

Comment (JanAckermann) [reply changelog/unreleased/38385:1]: Otherwise, good working solution :+1:

**Review by AlexAndBear — APPROVED** [review]

**Pull request merged**